dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java

```diff
@@ -3332,13 +3341,14 @@ private DynoDualWriterClient buildDynoDualWriterClient() {
 
         private DynoJedisClient buildDynoJedisClient() {
             DynoOPMonitor opMonitor = new DynoOPMonitor(appName);
+            ConnectionPoolMonitor cpMonitor = (this.cpMonitor == null) ? new DynoCPMonitor(appName) : this.cpMonitor;
```

Comment:
For the builder if not passed then we will use the default DynoCPMonitor, else the passed one.